src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -408,6 +416,37 @@ static void uct_ib_mlx5_wqe_dump(uct_ib_iface_t *iface, void *wqe, void *qstart,
     }
 }
 
+static void uct_ib_mlx5_resp_error_dump(uct_ib_iface_t *iface,
+                                        uct_ib_mlx5_err_cqe_t *ecqe,
+                                        char *buffer, size_t max)
+{
+    uct_rc_mlx5_iface_common_t *mlx5_iface = ucs_derived_of(iface,
+                                                            uct_rc_mlx5_iface_common_t);
+    char *ends                             = buffer + max;
+    uct_ib_mlx5_srq_t *srq                 = &mlx5_iface->rx.srq;
+    uct_ib_mlx5_srq_seg_t *seg             = uct_ib_mlx5_srq_get_wqe(srq,
+                                                                     ecqe->wqe_counter);
+    int i;
+
+    snprintf(buffer, ends - buffer, "strides %d%s next wqe %u desc %p",
```

Comment:
let's also print `seg->srq.ptr_mask` which shows which segments are reserved by the UCT user

Reply:
Added